gem_tasks/cucumber.rake

```diff
@@ -3,25 +3,8 @@
 require 'cucumber/rake/task'
 require 'cucumber/platform'
 
-class Cucumber::Rake::Task
```

Comment:
Have a look at the commit message when you have the time: https://github.com/mezuro/kolekti_metricfu/pull/15/commits/b76bb041d4679e082b537a6976e7bde8bcd6a7cf

It helps by making possible to reproduce the problem at first.